h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -42,36 +47,80 @@ function Label({
   );
 }
 
-export default function CreateGroupForm() {
-  const nameId = useId();
-  const descriptionId = useId();
+function TextField({
+  type,
+  limit,
+  label,
+  testid,
+  required = false,
+  autofocus = false,
+  classes = '',
+}: {
+  type: 'input' | 'textarea';
+  limit: number;
+  label: string;
+  testid: string;
+  required?: boolean;
+  autofocus?: boolean;
+  classes?: string;
+}) {
+  const id = useId();
+  const [value, setValue] = useState('');
+
+  function handleInput(e: InputEvent) {
```

Comment:
```suggestion
  const handleInput = (e: InputEvent) => {
```

This is just for stylistic consistency: Named functions at the top level of a module, arrow functions otherwise.

Reply:
Done